airbyte-webapp/src/pages/OnboardingPage/OnboardingPage.tsx

```diff
@@ -173,17 +166,10 @@ const OnboardingPage: React.FC = () => {
         );
 
         try {
-          if (!!destinations.length) {
```

Comment:
Since this seems to be the root place this was triggered, could you explain when this was triggered earlier, and why this is no longer happening in the app? I am not fully understanding the scenario where this happened and what the user-flow was when this was still in use.

Reply:
TBH I am also not sure when this was happening exactly, but as I remember in past it was possible to go back and forward during onboarding. So we were recreating destination in some cases. After this functionality was disabled - this endpoint was never called.